cSpell.json

```diff
@@ -14,7 +14,8 @@
         "wcag",
         "wdio",
         "webdriverio",
-        "webm"
+        "webm",
+        "Vidyard"
```

Comment:
Why did we add this?

Reply:
When I was trying to commit, it gave me a spell check error on "Vidyard" word. So I added it here.